dev/docker/hive/hive-site.xml

```diff
@@ -42,4 +42,25 @@
     <value>hdfs://__REPLACE__HOST_NAME:9000/user/hive/warehouse</value>
     <description>location of default database for the warehouse</description>
   </property>
+
+  <property>
+    <name>fs.s3a.access.key</name>
+    <value>S3_ACCESS_KEY_ID</value>
+  </property>
+
+  <property>
+    <name>fs.s3a.secret.key</name>
+    <value>S3_SECRET_KEY_ID</value>
+  </property>
+
+  <property>
+    <name>fs.s3a.endpoint</name>
+    <value>S3_ENDPOINT_ID</value>
+  </property>
+
+  <property>
+    <name>fs.s3a.aws.credentials.provider</name>
+    <value>org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider,com.amazonaws.auth.EnvironmentVariableCredentialsProvider,org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider</value>
+  </property>
+
```

Comment:
I remembered that we already merged this, why do you have this again in this PR?

Reply:
I should not be shown in the code changes as https://github.com/apache/gravitino/pull/4980 is merged, let me check the reason.

Reply:
@jerryshao Rebase the main branch solves the problem.